scripts/send_onetime_email_to_core_contributors.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+/*
+ * This script is useful for sending a one-time email to all group MEMBERS (aka core contributors).
+ */ 
+
+process.env.PORT = 3066;
+
+import _ from 'lodash';
+import moment from 'moment';
+import config from 'config';
+import Promise from 'bluebird';
+import debugLib from 'debug';
+import models, { sequelize } from '../server/models';
+import emailLib from '../server/lib/email';
+import roles from '../server/constants/roles';
+
+const debug = debugLib('onetime.email');
+
+const {
+  Group,
+  UserGroup,
+  User
+} = models;
+
+const init = () => {
+  console.log("\nStarting script to send a one-time email...\n");
+
+  const startTime = new Date;
+
+  // find all active groups
+  const groupQuery = {
+    attributes: ['id', 'slug'],
+    include: [ { model: models.Transaction, required: true }]
```

Comment:
That was intentional, I think we should tell all active collectives since this impacts all of them.